### DISCUSSION TITLE
FreeLoad.PolygonLoad introduce new coordinate system

### DISCUSSION THREAD

**Discussion by DannieCW** [Dlubal-Software/RFEM_Python_Client#275] [Ideas]
Dear developers, 

I am working on multiple polygon loads where the load direction has a rotation angle about Z-axis. Please see the picture below indicated by the red arrow. 

![image](https://github.com/Dlubal-Software/RFEM_Python_Client/assets/138588349/386001da-e81b-481d-858c-32d7f8a65967)

Given that the load direction cannot be specified directly with a rotation angle, I thought about creating a user-specified coordinates system with a rotation angle and then the load in the X direction. 

I can manually do so in the RFEM yet via FreeLoad.PolygonLoad I cannot find an option to introduce the new coordinates system. Would you like to advise me on how to realize what I want to achieve here?

Thanks in advance. 

Best regards,
Dannie

**Comment by OndraMichal** [comment]
Hi @DannieCW,
our PolygonLoad function doesn't have 'coordinate_system' but this attribute is still in the Web Services object itself. It can be set by parameter 'params' (params = {'coordinate_system' : 2}). That will do the trick. 'params' were added for exactly this purpose ... so we don't have to list all parameters in the functions.

**Reply by DannieCW** [reply]
Nice. That's a solution. Thanks!
